### PR TITLE
Doc: Add description of PEtab development process

### PR DESCRIPTION
**Pull request by dweindl**
As agreed on by the PEtab authors.

### CODE REVIEW

File: doc/development.rst [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+PEtab format development process
+++++++++++++++++++++++++++++++++
+
+We acknowledge that PEtab cannot accommodate everybody’s needs, but we are
+committed to addressing current and future requirements in upcoming versions of
+the PEtab format. Although we value backwards-compatibility, we don’t want to
+exclude breaking changes if justified by the benefits.
+
+Anybody is welcomed to propose changes or additions to PEtab. Any proposals
+shall be made using GitHub issues. Benefits, problems and potential
```
Comment (fbergmann) [inline doc/development.rst:161]: i would add a comma: "problems, and "

File: doc/development.rst [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+* Updated validator
+* Updated test suite
+* Updated changelog
+
+The PEtab editors jointly decide whether these requirements are met.
+
+Upon a new release, the PEtab editors ensure that
+
+* a new release is created in the GitHub repository
+* the new version of the specifications are deposited at Zenodo
```
Comment (fbergmann) [inline doc/development.rst:184]: should it be "the new version ... is deposited at zenodo" .. rather than are?

Comment (dweindl) [inline doc/development.rst:184]:
```suggestion
* the new version of the specifications is deposited at Zenodo
```

File: doc/development.rst (hunk 1 above)
Comment (dweindl) [inline doc/development.rst:161]:
```suggestion
shall be made using GitHub issues. Benefits, problems, and potential
```

File: doc/development.rst [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+coming from two different labs. To avoid a simultaneous replacement of all
+of the editors elected during the first election, the first election will be
+subject to the following special rules: 2 persons will be elected for 3
+years, 2 persons for 2 years and one person for 1 year. The persons with
+more votes will be elected for a longer period (assuming that they accept
+for the longer period). In case of an equal number of votes among any of the
+top 5 candidates, there will be run-off elections between those candidates
+with equal numbers of votes. The editors-elect are given 3 working days to
+accept the election.
+
```
Comment (svenjakemmer) [inline doc/development.rst:151]: In the email announcing the editors we had the phrase: "(1 year, with the option of +1 year if one of the above editors hands over his responsibility)"
Should we include this in the document? Something like
```suggestion
If an editor decides to hand over his editorial role before his election period expires, editors elected for a shorter time period can decide to take over and expand their time as an editor for the remaining time.
```

**Review by svenjakemmer — CHANGES_REQUESTED** [review]

File: doc/development.rst [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+The first election will be held among the authors of the original PEtab
+publication. Nominees are not required to be among the authors. The election
+is to be managed by two persons who are not among the candidates and are
+coming from two different labs. To avoid a simultaneous replacement of all
+of the editors elected during the first election, the first election will be
+subject to the following special rules: 2 persons will be elected for 3
+years, 2 persons for 2 years and one person for 1 year. The persons with
+more votes will be elected for a longer period (assuming that they accept
+for the longer period). In case of an equal number of votes among any of the
+top 5 candidates, there will be run-off elections between those candidates
```
Comment (FFroehlich) [inline doc/development.rst:148]:
```suggestion
top 5 candidates, there would have been run-off elections between those candidates
```

File: doc/development.rst [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+with equal numbers of votes. The editors-elect are given 3 working days to
```
Comment (FFroehlich) [inline doc/development.rst:149]:
```suggestion
with equal numbers of votes. The editors-elect were given 3 working days to
```

**Review by FFroehlich — APPROVED** [review]

**Comment by FFroehlich** [comment]
I think first special election is something that already happened and will not apply to future events. Accordingly, this should be  described in past tense.

File: doc/development.rst [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+  organized by the PEtab editors as soon as possible. Voters will again have a
+  number of votes equal to the number of vacant positions.
+* If the editor-elect accepts, the other editors announce the new editor on the
+  PEtab mailing list. The editors shall furthermore announce the number of
+  votes each nominee, elected or not, has received.
+
+Special procedure for the first election (February, 2021):
+----------------------------------------------------------
+
+The first election was be held among the authors of the original PEtab
```
Comment (FFroehlich) [inline doc/development.rst:144]:
```suggestion
The first election was held among the authors of the original PEtab
```

**Review by svenjakemmer — APPROVED** [review]
Sounds good to me now :)

**Pull request merged**